iphone/Classes/TiUIWindowProxy.m

```diff
@@ -937,6 +940,49 @@ - (void)cleanupWindowDecorations
     [barImageView removeFromSuperview];
   }
 }
+
+- (void)processForSafeArea
```

Comment:
Should be compiled out by `#if IS_XCODE_9` to decrease the code-size for Xcode < 9.